index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="style.css"/>
+	<link rel="preconnect" href="https://fonts.gstatic.com"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet"/>
+</head>
+<body class="background">
+	<header>
+		<nav>
+			<a href="#home">home.</a>
+			<a href="#projects">projects.</a>
+			<a href="#about">about.</a>
+			<a href="#contact">contact.</a>
+		</nav>
+	</header>
+	<main id="home">
+		<div class="top" id="fade">
+		<h1>maria</h1>
+		<h1>gusova.</h1>
+	</div>
+	<div class="bottom" id="fade">
+		<h2>software</h2>
+		<h2>developer.</h2>
+	</div>
+	</main>
+	<section id="projects" class="projects">
+		<h3>projects.</h3>
+		<div class="grid">
+			<div class="project">
+				<a href="https://github.com/firefox-devtools/profiler"><img src="profiler.png" alt="firefox profiler"/><p>Firefox Profiler</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/de-tough/"><img src="detough.png" alt="DE-Tough"/><p>DE-Tough LTD</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/grumpygit/"><img src="grumpygit.png" alt="Grumpy Git"/><p>Grumpy Git</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/pomodoro-clock/"><img src="pomodoro.png" alt="Pomodoro Clock"/><p>Pomodoro Clock</p></a>
```

Comment:
I'd also suggest making these links open in a new tab using `target="_blank"` on the a tags